### PR TITLE
rn-335 update permission groups for covid samoa visualisations

### PR DESCRIPTION
**Pull request by avaek**
### Issue #: https://linear.app/bes/issue/RN-335/permissions-issues-in-the-covid-19-samoa-project

This migration makes the permission group associated with specific reports match the permission group associated with their relevant dashboard items / map overlays

### CODE REVIEW

File: packages/database/src/migrations/20220210214956-UpdateCovid19SamoaVisualisationPermissions-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  const selectNewPermissionId = await db.runSql(`
```
Comment (chris-pollard) [inline packages/database/src/migrations/20220210214956-UpdateCovid19SamoaVisualisationPermissions-modifies-data.js:28]: You could use codeToId() from '../utilities'

**Comment by chris-pollard** [comment]
Looks good to me!

**Review by chris-pollard — APPROVED** [review]

**Comment by nick-beyondessential** [comment]
@chris-pollard, @avaek  Do we want to do the same for UNFPA?

**Comment by avaek** [comment]
@nick-beyondessential - After dev chat or on Monday I'll make a separate card for the UNFPA issue we discussed today on Slack and talk to Erin about it to find exactly which permissions should be used there

**Pull request merged**